packages/tabs/__tests__/integrations.js

```diff
@@ -43,13 +43,13 @@ const init = (mode) => {
         </section>
     </div>`;
 
-    TabSet = tabs('[role=tablist]', {activation: activation});
+    TabSet = tabs('[role=tablist]', { activation });
 };
 
 describe(`Tabs > Initialisation`, () => {
     
-    beforeAll(() => {init()});
-    console.log(TabSet)
+    beforeAll(() => {init();});
+    console.log(TabSet);
```

Comment:
This could probably just come out?  I bet I most likely left that in while learning Jest!